tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -0,0 +1,37 @@
+*** Settings ***
```

Comment:
Are you sure `tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot` is the right place to locate this test case? it seems to be something related more to Jupyter Test Suite

Reply:
It seems that this test is about the cuda build is successfully loaded after installation and to verify if is working fine, he's checking that it's possible to spawn a notebook. Maybe it could be more suitable in tests/Tests/100__deploy/140__builds/140__builds.robot what do you think?

Reply:
hm I think it is about checking Pytorch and Tensorflow images as reported https://issues.redhat.com/browse/RHODS-1469

Reply:
Okay, But it isn't specific to pytorch or tensorflow. Is it okay If I move it test-jupyterlab-notebook.robot? @bdattoma @pablofelix

Reply:
If this is about checking that the builds were successful, shouldn't it go into `tests/Tests/100__deploy/100__installation/102__post_install.robot`?

Reply:
Yes, But it isnt clear if test is about verifying builds or To check if we can  spawn Pytorch/tensorflow.

Reply:
I am not sure what test you're trying to automate. If you're just trying to get coverage for this old bug (https://issues.redhat.com/browse/RHODS-1469), then we already have tests that spawn tensorflow and pytorch, so I don't think it's needed?

Reply:
This test try to automate a verification that is needed as part of the build process to verify that everything is working as it is supposed.

Reply:
Is it okay if I move it to post_install.robot @pablofelix @bdattoma @lugi0 ?

Reply:
I think that's fine